contents/docs/libraries/ios/index.mdx

```diff
@@ -140,35 +140,158 @@ This will remove the Super Property and subsequent events will not include it.
 
 If you are doing this as part of a user logging out you can instead simply use `PostHogSDK.shared.reset` which takes care of clearing all stored Super Properties and more.
 
-## Amending event properties before they are captured
+## Amending, dropping or sampling events
```

Comment:
i think this section misses warning users about sampling events that would cause some issues such as `identify`, `snapshot`, etc.